### ISSUE TITLE
Merge the English and Russian versions?

### ISSUE THREAD

**Issue by pharpend** [gazay/lysa#5]
Hi, main-ish author here.

It occurs to me that, in the future, there will likely be original content added
to the Russian version of LYSA, which we'll want to translate into
English. There may also be a French version sometime soon. With that in mind, it
will quickly be a pain to sync everything between the various
repositories. 

So, let's put everything in one repo, but with different directories for
different languages. That is,

    lysa/
        README.md
        LICENSE
        docs/
        en/
            README.md
            appendices/
            bibliographies/
            ...
        ru/
            README.md
            appendices/
            bibliographies/
            ...
        fr/
            README.md
            appendices/
            bibliographies/
            ...
            
I wouldn't want multiple people having commit access to the central repository,
because that gets ugly very quickly, with politics and such. I would also not
want to have 3 different active branches in the central repository, because that
would inevitably cause politics. (Trust me, I tried this with
https://github.com/learnmath/lysa . Politics everywhere.) I think the best
option is for y'all to keep your own repo, and then we pull-request them
periodically.

So, to summarize, here's what I'm proposing:

* I want to take your changes, and put them in a new `ru/` directory in the
  central repository.

* I will put the English version in the `en/` directory.

* You will keep this repo, but we'll sync them periodically.

Is that alright with everyone?

**Comment by gazay** [comment]
Yep, it's ok for me. Scheme is the same as in https://github.com/ruby/www.ruby-lang.org. But don't you still think that better be working on original book in github? Here much more people who can find and help with the book, also here much more done for easy collaboration. Syntax highlighting, transparent commenting on PR lines, PRs. I think you can win more (stars, pr, small orphographic fixes etc) than lose if you move repo to github.com/lysa/lysa for example. For example I found small mistake (as I think it is a mistake), but it is so small and I was uncomfortable to add my ssh key to gitlab, dig into gitlab's interface, learn how to make PR there just for this small fix. I think as most of IT people now here it will be same story for them.

**Comment by pharpend** [comment]
Yeah, you are right, unfortunately.

**Comment by pharpend** [comment]
Okay, I merged them at https://github.com/learnyou/lysa . Pull from the master branch (I promise it won't delete any of your stuff).

**Comment by gazay** [comment]
Great!
